inference-engine/src/vpu/graph_transformer/include/vpu/stage_builder.hpp

```diff
@@ -312,6 +312,15 @@ class StageBuilder final {
             const Data& input0,
             const Data& input1,
             const Data& output);
+
+    Stage addGatherNDStage(
```

Comment:
Where is the implementation? 
If it is not required now, please, remove the declaration